examples/02-HFSS/HFSS_eigenmode.py

```diff
@@ -0,0 +1,147 @@
+"""
+HFSS: Eigenmode Filter
```

Comment:
```suggestion
HFSS: Eigenmode filter
```
Goggle developer style guide uses sentence case (not title case)